models/balancer/arbitrum/balancer_arbitrum_schema.yml

```diff
@@ -68,54 +69,32 @@ models:
             - evt_tx_hash
             - evt_index
             - block_date
-      - check_seed_legacy:
```

Comment:
do you have any particular reasoning to remove tests? both here and the unique ones?

i'm okay with the removal ultimately, but curious the reasoning behind it. if it's difficulty migrating the test, we have a process down to help migrate seeds and tests.

if you ultimately want to remove this still, i'd say go ahead and remove the seed file itself as well

Reply:
The removal of the test was due to the difficulty migrating the tests and addressing the errors returned durning the migrate process. What is the process you all have?

Reply:
since all blockchain spells are in this PR, you'll be safe just to add here in same PR. in other scenarios, if migrating just one blockchain for instance, it'd be easier to do in separate PR to avoid seed tests running on chains not included. anyway, with that in mind, you can add a schema file for the seed in the same directory, like [here](https://github.com/duneanalytics/spellbook/pull/3886/files#diff-ed19154a4e0a755b19a400b89b091cd7b2c4b3a18d39969a9dbe3d913f1bcda1). in this file, you can force columns to a certain datatype to match your spell datatypes, most importantly the join condition ones since the join would fail if they differ

then you can force a change to your seed file itself, so it appears in the PR and builds in CI process.

the check seed macro * should * be fine as-is, you can leave that out for now